modules/cluster-logging-log-forwarding-about.adoc

```diff
@@ -9,17 +9,19 @@ The {product-title} cluster log forwarding feature uses a combination of _output
 
 [NOTE]
 ====
-If you want to use only the default internal {product-title} Elasticsearch instance, do not configure any outputs and pipelines. 
+If you want to use only the default internal {product-title} Elasticsearch logstore, do not configure the Log Forwarding feature. 
 ==== 
 
-An output is the destination for log data and a pipeline defines simple routing for one source to one or more outputs. 
+By default, the Cluster Logging Operator sends logs to the default internal Elasticsearch logstore, as defined in the Cluster Logging Custom Resource. To use the Log Forwarding feature, you create a custom `logforwarding` configuration file to send logs to endpoints you specify.  
```

Comment:
I don't think we've been internally consistent with our usage of custom resource, partially because upstream and developer usage may be inconsistent.

In any case, a custom resource is a resource defined using k8s CustomResourceDefinition object. So that's where _custom resource_, and thus _CR_ comes from. I don't think it should be capitalized. It's doubly confusing because developers will abbreviate resources, such as CustomResourceDefinition or HorizontalPodAutoscaler as CRD or HPA, but they'r really resource names... So it's different than abbreviating, say, RAM, which is really a proper noun, abbreviated.

Sigh.

So;

s/Custom Resource/custom resource/

Reply:
I'm going to leave this one for now, if you don't mind. it would require 44 changes in 28 files (per Atom) to make `Cluster Logging custom resource` consistent. I can change these all in a single sweep in a separate PR.

Reply:
@mburke5678, oh, that's interesting; Usually it's [Object] CR. This is _feature name_ CR? Or is there a ClusterLogging object?

Reply:
@jboxman It's a custom resource that the Cluster Logging Operator uses when creating the various cluster logging components (Elasticearch, Kibana, Fluentd, Curator, etc.). It's `kind: "ClusterLogging"` and named _instance_. 
Probably should refer to it as the _ClusterLogging CR_?

Reply:
@mburke5678 probably for now. I wanted to devise a convention for this stuff, but it's intractable. It would be nice to be able to write about _cluster logging_, and mention the ClusterLogging CR when directly interacting with it. But that approach might not stand up to scrutiny.